### PR TITLE
[Fortran] Rank and total size

### PR DESCRIPTION
**Pull request by chovyy**
Adds the functions fs_get_rank and fs_get_total_size to the Fortran interface.

### CODE REVIEW

**Comment by havogt** [comment]
launch jenkins

File: src/serialbox-c/FortranWrapper.cpp [hunk 1]
```diff
@@ -172,6 +172,13 @@ void serialboxFortranLoc(const void* basePtr, intptr_t* loc)
 	*loc = (long) basePtr;
 }
 
+void serialboxFortranSerializerGetFieldRank(const void* serializer, const char* name, int* rank) {
+
+	const Serializer* ser = toConstSerializer(static_cast<const serialboxSerializer_t*>(serializer));
+
+    *rank = ser->getFieldMetainfoImplOf(name).dims().size();
```
Comment (clementval) [inline src/serialbox-c/FortranWrapper.cpp]: I think you have similar indent problem in this PR

**Comment by chovyy** [comment]
Fixed, but if you accept PR #142, we can just forget this one.

**Comment by havogt** [comment]
Merged with #142

**Pull request closed without merging**